updates/next.json

```diff
@@ -45,19 +45,19 @@
       }
     },
     {
-      "version": "35.20220227.1.1",
+      "version": "35.20220313.1.0",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "35.20220313.1.0",
+      "version": "36.20220325.1.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 2880,
-          "start_epoch": 1647439200,
+          "start_epoch": 1648562400,
           "start_percentage": 0.0
```

Comment:
we need to add a barrier here - look at https://github.com/coreos/fedora-coreos-streams/commit/f1969b2 for an example